cookiecutter-django-ida/{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+
+**Merge checklist:**
+Check off if complete *or* not applicable:
+- [ ] Documentation updated (not only docstrings)
+- [ ] Commits are squashed
```

Comment:
```suggestion
- [ ] Fixup commits are squashed away
```
If I were a new dev, I think I'd read this as "always squash into one big commit".

What I think we want to say here is "you can have one big commit, or you can have a handful of smaller commits that make sense individually, but don't merge a bunch of fixup commits". I've tried to condense that sentiment into a snappy checklist item.

Reply:
Ah yeah, good wording.